### PR TITLE
Update EIP-2481: Move to final

### PR DESCRIPTION
**Pull request by MariusVanDerWijden**
EIP has been in last call for ~6 months. Eth/66 has long been implemented by all clients, so this should be final already

### CODE REVIEW

**Review by cburgdorf — DISMISSED** [review]

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Walidator errors need to be fixed (except the discussions-to)

File: EIPS/eip-2481.md [hunk 1]
```diff
@@ -1,20 +1,17 @@
 ---
 eip: 2481
-title: "eth/66: request identifier"
+title: "eth/66 request identifier"
```
Comment (SamWilsn) [inline EIPS/eip-2481.md:3]:
```suggestion
title: eth/66 request identifier
```

File: EIPS/eip-2481.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
 author: Christoph Burgdorf (@cburgdorf)
```
Comment (SamWilsn) [inline EIPS/eip-2481.md:4]:
```suggestion
```

File: EIPS/eip-2481.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
-discussions-to: https://github.com/ethereum/EIPs/issues/2482
-status: Last Call
+description: Introduces a request id for all requests of the eth protocol
```
Comment (SamWilsn) [inline EIPS/eip-2481.md:5]:
```suggestion
description: Introduces a request id for all requests of the eth protocol
author: Christoph Burgdorf (@cburgdorf)
```

File: EIPS/eip-2481.md [hunk 4]
```diff
@@ -1,20 +1,17 @@
...
+discussions-to: https://ethereum-magicians.org/t/eip-2481-eth-66-request-identifiers/12132
+status: Final
 last-call-deadline: 2021-08-11
```
Comment (SamWilsn) [inline EIPS/eip-2481.md:8]:
```suggestion
```

File: EIPS/eip-2481.md [hunk 5]
```diff
@@ -1,20 +1,17 @@
...
 requires: 2464
 ---
 
-## Simple Summary
-
-This document proposes a way to increase the efficiency of the `eth` networking protocol while at the same time reducing complexity in Ethereum node implementations. It does so by introducing a request id to all requests which their corresponding responses must include.
-
 ## Abstract
 
 The `eth` protocol defines various request and response commands that are used to exchange data between Ethereum nodes. For example, to ask a peer node for a specific set of headers, a node sends it the [`GetBlockHeaders`](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#getblockheaders-0x03) command.
```
Comment (SamWilsn) [inline EIPS/eip-2481.md:17]:
```suggestion
The `eth` protocol defines various request and response commands that are used to exchange data between Ethereum nodes. For example, to ask a peer node for a specific set of headers, a node sends it the [`GetBlockHeaders`](https://github.com/ethereum/devp2p/blob/40ab248bf7e017e83cc9812a4e048446709623e8/caps/eth.md#getblockheaders-0x03) command.
```

**Review by SamWilsn — CHANGES_REQUESTED** [review]
Couple changes you need to make that were too far from a changed line for me to comment on:

 - Links to devp2p are allowed, but have to be qualified by a commit hash. See the one suggestion I was able to make for the format. (needs #6091)
 - "Test cases" -> "Test Cases"

**Review by SamWilsn — APPROVED** [review]

**Comment by SamWilsn** [comment]
@cburgdorf would you mind reapproving if everything is in order?

**Review by cburgdorf — APPROVED** [review]

**Pull request merged**